src/network/ConnectionManager.ts

```diff
@@ -214,7 +214,7 @@ export class ConnectionRegistryImpl implements ConnectionRegistry {
             if (dataMember) {
                 const member = this.clusterService.getMember(uuid);
                 if (!member || member.liteMember)
-                    continue;
+                    {continue;}
```

Comment:
can you go over this commit and fix the weird-looking if statements such as this. (curly braces on newline etc.)